src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -932,12 +946,15 @@ public PreferenceSettings()
             BackupFiles = new List<string>();
             BackupLocation = string.Empty;
 
+            TemplateFilePath = string.Empty;
+
             LibraryZoomScale = 100;
             PythonScriptZoomScale = 100;
 
             CustomPackageFolders = new List<string>();
 
             PythonTemplateFilePath = "";
+            PythonTemplateFilePath = PathManager.TemplateDirectoryName;
```

Comment:
This line is intended? value for `PathManager.TemplateDirectoryName` is "templates" right?